engine-vuex/src/wwtaware.ts

```diff
@@ -294,6 +294,8 @@ export class WWTAwareComponent extends Vue {
         "toggleTourPlayPauseState",
         "updateTableLayer",
         "zoom",
+        "move",
+        "tilt"
```

Comment:
Very minor stylistic thing: I think it is good to get into the habit of using trailing commas whenever possible, because it makes copy-pasting easier and reduces the noise on diffs,

Reply:
(I didn't intentionally end that comments with a comma but it's so apropos that I'll leave it there!)

Reply:
I'd never considered the effect that this would have on diffs- I'll add in the comma